task_2/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Form Validtion</title>
+</head>
+<body>
+    <div class="wrapper">
+        <div class="header">
+            <div class="header-title">
+                <h1>Create Account</h1>
+            </div>
+        </div>
+        <div class="main">
+            <form action="" onsubmit="checkValidate(event)">
+                <div class="form-field">
+                    <input type="text" class="input-field" id="email" placeholder="Email Address" />
+                    <p id="email-error"></p>
+                </div>
+                <div class="form-field">
+                    <input type="text" class="input-field" id="company-name" placeholder="Company Name" />
+                    <p id="company-error"></p>
+                </div>
+                <div class="form-field">
+                    <input type="password" class="input-field" id="password" placeholder="Password" minlength="8" />
+                    <p id="password-error"></p>
+                </div>
+                <div class="form-field">
+                    <input type="password" class="input-field" id="password-check" placeholder="Confirm password" />
+                    <p id="password-check-error"></p>
+                </div>
+                <div class="form-field">
+                    <select id="city" class="input-field selector-field">
+                        <option default disabled selected hidden class="selector-field">Choose Your City</option>
+                        <option value="Jordan">Jordan</option>
+                        <option value="Japan">Japan</option>
+                        <option value="America">America</option>
+                        <option value="Oman">Oman</option>
+                    </select>
+                    <p id="option-error"></p>
+                </div>
+                <div class="form-field">
+                    <button type="submit" class="input-field submit-field">CREATE ACCOUNT</button>
```

Comment:
Use css     `    text-transform: uppercase;
`
Instead of writing it in upper case 
```suggestion
                    <button type="submit" class="input-field submit-field">create account </button>
```